3582/3/db/migrate/20190729172137_create_restaraunts.rb

```diff
@@ -0,0 +1,10 @@
+# :reek:FeatureEnvy :reek:UncommunicativeVariableName
+class CreateRestaraunts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :restaraunts do |t|
```

Comment:
Please, figure it out. There should be no grammatical errors in the names of tables, models, controllers...